src/common/circle-series.component.ts

```diff
@@ -44,7 +44,7 @@ import { id } from '../utils/id';
         ngx-tooltip
         [tooltipPlacement]="'top'"
         [tooltipType]="'tooltip'"
-        [tooltipTitle]="getTooltipText(circle)"
+        [tooltipTitle]="getTooltipText(circle, circle.min, circle.max)"
```

Comment:
Done. Along the way, I learned something new about JavaScript and TypeScript :).